matcha-app/Matcha
# Origin/view support

### What does this accomplish?
Adds core manipulations on the `View` class, sets up basic infrastructure on the `EditText` and `TextView` classes
Fixes crashing when malformed JSON is inputted
UI Updates on web
 - We now use Boostrap for some nicer formatting
 - Added a logo
### What is the scope of this change?
Everything

### How can I test this?
```
{ 
	type:"TextView",
	text:"Matcha", 
	textColor:"#000000", 
	textSize:"24", 
	padding:"49", 
	layoutWidth:"match_parent",
	layoutHeight:"wrap_content",
	textAlignment:"viewStart"
}
```

```
{ 
	type:"EditText",
	text:"Matcha", 
	textColor:"#000000", 
	textSize:"24", 
	padding:"49", 
	layoutWidth:"match_parent",
	layoutHeight:"wrap_content",
	textAlignment:"viewStart"
}
```
Malform the JSON and an error message should show on the phone
- [x] This build compiles
- [x] I have tested this and AFAIK these changes do not add any additional crashes/bugs
